Controller/ConfigController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace BestSellers\Controller;
+
+
+use BestSellers\Form\Configuration;
+use BestSellers\BestSellers;
+use Thelia\Controller\Admin\BaseAdminController;
+use Thelia\Model\ConfigQuery;
+use TheliaSmarty\Template\Plugins\Form;
+
+class ConfigController extends BaseAdminController
+{
+    public function setAction()
+    {
+        $form = new Configuration($this->getRequest());
+        $response = null;
+
+        $configForm = $this->validateForm($form);
+        BestSellers::setConfigValue('order_types', $configForm->get('order')->getData(),null, true);
+
```

Comment:
It seems a little bit dangerous to store the data entered by the admin without filtering or checking.

Reply:
This is now encapsulated in a try/catch.